### PR TITLE
feat: add context value descriptions

### PR DESCRIPTION
**Pull request by olav**

### CODE REVIEW

**Review by chriswk — COMMENTED** [review]
This is mostly used for validation on the frontend?

**Comment by chriswk** [comment]
As long as we're not breaking our client SDKs, this seems good

**Review by chriswk — APPROVED** [review]
LGTM

**Comment by olav** [comment]
> This is mostly used for validation on the frontend?

Yep! Just an Admin API thing.

File: src/migrations/20220411103724-add-legal-value-description.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        -- 1. Convert '1,2,3'::text to "1,2,3"::json
+        alter table context_fields
```
Comment (FredrikOseberg) [inline src/migrations/20220411103724-add-legal-value-description.js:7]: Would prefer if we follow the convention in previous migrations of uppercasing the ALTER TABLE / SELECT statements here. 

As for the migration itself, I _think_ it looks good, but I'd like @chriswk and @sighphyre  to weigh in here.

**Review by FredrikOseberg — CHANGES_REQUESTED** [review]
Close to done I think.

File: src/migrations/20220411103724-add-legal-value-description.js (hunk 1 above)
Comment (chriswk) [reply src/migrations/20220411103724-add-legal-value-description.js]: The SQL seems sane, good catch on the casing to follow convention

**Review by FredrikOseberg — APPROVED** [review]
LGTM

**Pull request merged**